ansible/roles/nginx/vars/couchdb2.yml

```diff
@@ -18,6 +18,8 @@ nginx_sites:
        balancer: "{{ deploy_env }}_couchdb_{{ couchdb2_proxy_port }}"
        proxy_redirect: ' off'
        client_max_body_size: ' 10m'
+       client_body_buffer_size: ' 512k'
+       proxy_max_temp_file_size: ' 0'
```

Comment:
I don't think that they're required. I started this file by copying another file and it had a space there. i've been meaning to go back and make sure they aren't required